projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo.php

```diff
@@ -91,6 +91,13 @@ public function set_custom_og_tags( $tags ) {
 		$post_custom_description = Jetpack_SEO_Posts::get_post_custom_description( get_post() );
 		$front_page_meta         = Jetpack_SEO_Utils::get_front_page_meta_description();
 
+		if ( class_exists( 'woocommerce' ) && is_post_type_archive( 'product' ) ) {
```

Comment:
I imagine as well you could change this one line to add the WooCommerce [is_shop](https://woocommerce.wp-a2z.org/oik_api/is_shop/) function which is similar but covers an additional scenario?

```suggestion
		if ( class_exists( 'woocommerce' ) && is_shop() )  {
```